experiments/ClimaEarth/run_amip.jl

```diff
@@ -856,6 +856,53 @@ The postprocessing includes:
 - Error against observations
 - Optional additional atmosphere diagnostics plots
 - Plots of useful coupler and component model fields for debugging
+
+### Add a new variable to compare against observations
```

Comment:
The comments in this section are really helpful! Do you think you could move them to the docs? We should add a new section `Model Output` (or something like that) and add a page about the leaderboard that contains this information.

The AMIP driver (run_amip.jl) is pretty complex right now but we're moving toward slimming it down, so it would be better to move a big comment like this to the docs